buildengine/build_go_test.go

```diff
@@ -197,3 +198,38 @@ func TestExternalType(t *testing.T) {
 	err = os.RemoveAll(buildDir)
 	assert.NoError(t, err, "Error removing build directory")
 }
+
+func TestBuildChecksMinFTLVersion(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+	moduleDir := "testdata/projects/another"
+	buildDir := "_ftl"
+
+	ctx := log.ContextWithLogger(context.Background(), log.Configure(os.Stderr, log.Config{}))
+	module, err := LoadModule(moduleDir)
+	assert.NoError(t, err)
+
+	tests := []struct {
+		v       string
+		wantErr bool
+	}{
+		{"dev", false},
+		{"1.3.0", false},
+		{"0.129.2", true},
+	}
+
+	for _, test := range tests {
+		sch := &schema.Schema{}
+		ftl.Version = test.v
+		err = Build(ctx, sch, module)
+		if test.wantErr {
+			assert.Error(t, err, "when version=%v, we expected an error but did not receive one", test.v)
+		} else {
+			assert.NoError(t, err, "when version=%v, we received an unexpected error", test.v)
+		}
+	}
+
+	err = os.RemoveAll(buildDir)
```

Comment:
Do this in a `t.Cleanup()` function, and ignore the error if it's an `os.ErrNotExist`

Reply:
I ended up moving all the build changes to the module loading code instead after talking to Lizzy (to improve the testability) but I'll keep this in mind for future test cleanup - thanks for the tip!